tests/integration/test_vulnerability_detector/test_providers/test_update_from_year.py

```diff
@@ -196,7 +196,7 @@ def test_update_from_year_extended(configuration, metadata, set_wazuh_configurat
         evm.check_nvd_download_log(update_year)
         evm.check_nvd_download_completion(update_year)
 
-        evm.check_provider_database_update_finish_log(provider_name)
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=None, provider_name=provider_name)
```

Comment:
```suggestion
        evm.check_provider_database_update_finish_log(provider_name=provider_name)
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800